tidb-cloud/import-csv-files.md

```diff
@@ -14,44 +14,61 @@ This document describes how to import uncompressed CSV files from Amazon Simple
 
 ## Step 1. Prepare the CSV files
 
-1. If a CSV file is larger than 256 MB, consider splitting the file into smaller files, each with a size around 256 MB.
+1. If a CSV file is larger than 256 MB, consider splitting it into smaller files, each with a size around 256 MB.
 
-    TiDB Cloud supports importing very large CSV files but performs best with multiple input files around 256 MB in size. This is because TiDB Cloud can process multiple files in parallel which can greatly improve the import speed.
+    TiDB Cloud supports importing very large CSV files but performs best with multiple input files around 256 MB in size. This is because TiDB Cloud can process multiple files in parallel, which can greatly improve the import speed.
 
-2. According to the naming convention of existing objects in your bucket, identify a text pattern that matches the names of the CSV files to be imported.
+2. Name the CSV files as follows:
 
-    For example, to import all data files in a bucket, you can use the wildcard symbol `*` or `*.csv` as a pattern. Similarly, to import the subset of data files in partition `station=402260`, you can use `*station=402260*` as a pattern. Make a note of this pattern as you will need to provide it to TiDB Cloud in [Step 4](#step-4-import-csv-files-to-tidb-cloud).
+    - If a CSV file contains all data of an entire table, name the file in the `${db_name}.${table_name}.csv` format, which maps to the `${db_name}.${table_name}` table when you import the data.
+    - If the data of one table is separated into multiple CSV files, append a numeric suffix to these CSV files. For example, `${db_name}.${table_name}.000001.csv` and `${db_name}.${table_name}.000002.csv`. The numeric suffixes can be inconsecutive but must be in ascending order. You also need to add extra zeros before the number to ensure all the suffixes are in the same length.
 
-## Step 2. Create the target table schema
+    > **Note:**
+    >
+    > If you cannot update the CSV filenames according to the preceding rules in some cases (for example, the CSV file links are also used by your other programs), you can keep the filenames unchanged and use the **Custom Pattern** in [Step 4](#step-4-import-csv-files-to-tidb-cloud) to import multiple CSV files to a single table.
```

Comment:
This should be "If you cannot update the CSV filenames according to the preceding rules in some cases (for example, the CSV file links are also used by your other programs ）, or to import multiple CSV files to a single table. You can use the **Custom Pattern** in [Step 4](#step-4-import-csv-files-to-tidb-cloud) “

Reply:
I think that even if users do not use Custom Pattern, they can still use DB/Table filters to import multiple CSV files to a single table, as long as their CSV files use the same table name.